### PR TITLE
[896, 902, 906] Food & Fluid

### PR DESCRIPTION
**Pull request by moderatemisbehaviour**
- Users can perform food & fluid observations.
  - Tables in mobile, desktop, and reports.
  - Observation form in mobile with reference charts.
- Some new conventions for the front-end to request information it needs from observation models.
  - `an_obs_model.get_submission_message()` provides the message displayed in the body of the __successfully submitted__ pop up on the mobile UI.
  - `get_formatted_obs` provides a dictionary of observation data formatted for display on the front-end.

### CODE REVIEW

File: nh_eobs_mental_health/models/nh_eobs_api.py [hunk 1]
```diff
@@ -12,6 +12,44 @@ class NHeObsAPI(orm.AbstractModel):
...
+        },
+        {
+            'type': 'gcs',
+            'name': 'Glasgow Coma Scale (GCS)'
+        },
+        {
+            'type': 'pbp',
+            'name': 'Postural Blood Pressure'
+        }
+    ]
```
Comment (moderatemisbehaviour) [inline nh_eobs_mental_health/models/nh_eobs_api.py]: Replace this and other `get_active_observation` method with one generic implementation based on the `get_data_visualisation_resources` one.

File: nh_observations/observations.py [hunk 2]
```diff
@@ -271,38 +285,31 @@ def read(self, cr, uid, ids, fields=None, context=None,
...
+        return obs
 
-        :param args:
-        :param kwargs:
-        :return:
-        :rtype: dict
-        """
-        return self.read(*args, **kwargs)
+    @api.multi
+    def read_labels(self, fields=None, load='_classic_read'):
```
Comment (moderatemisbehaviour) [inline nh_observations/observations.py]: Add docstrings to this method and others.

File: nh_food_and_fluid/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+nurse_access_nh_clinical_dietary_need,nurse:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_nurse,1,1,1,1,1
+manager_access_nh_clinical_dietary_need,manager:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_ward_manager,1,1,1,1,1
+senior_manager_access_nh_clinical_dietary_need,senior_manager:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_senior_manager,1,1,1,1,1
+dev_access_nh_clinical_dietary_need,developer:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_dev,1,1,1,1,1
+admin_access_nh_clinical_dietary_need,admin:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_admin,1,1,1,1,1
+adt_access_nh_clinical_dietary_need,adt:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_adt,1,1,1,1,1
+hca_access_nh_clinical_dietary_need,hca:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_hca,1,1,1,1,1
+doctor_access_nh_clinical_dietary_need,doctor:access_nh_clinical_dietary_need,model_nh_clinical_dietary_need,nh_clinical.group_nhc_doctor,1,1,1,1,1
```
Comment (moderatemisbehaviour) [inline nh_food_and_fluid/security/ir.model.access.csv]: Copy `nh_neurological` access rules.

**Comment by moderatemisbehaviour** [comment]
LGTM :shipit:

**Comment by moderatemisbehaviour** [comment]
LGTM :shipit:

**Comment by Nayira** [comment]
Test cases for EOBS-902, EOBS-906 and EOBS-896 are a Pass
Following feature branch bugs verified as Fixed: EOBS-1052, EOBS-1054, EOBS-1055, EOBS-1057,EOBS-1061, EOBS-1062, EOBS-1063, EOBS-1064, EOBS-1065, EOBS-1066, EOBS-1069, EOBS-1070, EOBS-1071, EOBS-1078, EOBS-1079, EOBS-1080, EOBS-1083, EOBS-1084, EOBS-1085 and EOBS-1086 

EOBS-1056 and EOBS-1068 partially fixed. Agreed to fix with Weight obs. :shipit:  already! :fireworks:

**Pull request merged**